src/emoji/__tests__/data-test.js

```diff
@@ -1,64 +1,89 @@
-import { getFilteredEmojiNames, nameToEmojiMap } from '../data';
+/* @flow strict-local */
```

Comment:
Thanks for this upgrade!

Like I asked earlier, I really do want this *before* the commits that make substantive changes to these tests, though. :slightly_smiling_face: A major reason I want to have it is so we know the tests are well-typed through each of those changes -- that helps in reading them.